bhs_api/item.py

```diff
@@ -189,6 +189,15 @@ def enrich_item(item, db=None, collection_name=None):
     if pictures:
         main_image_id = None
         for image in pictures:
+            # Add 'PictureUrl' to all images of an item
+            image_id = image['PictureId']
```

Comment:
this is what happened on test environment, worth to look into it - why there are images without PictureId